README.md

```diff
@@ -136,22 +136,22 @@ The continued development and maintenance of GitHub1s is made possible by these
 
 - [febaoshan/edge-extensions-github-code-viewer](https://github.com/febaoshan/edge-extensions-github-code-viewer) - [Chrome Web Store](https://chrome.google.com/webstore/detail/github-code-viewer/ecddapgifccgblebfibdgkagfbdagjfn)
 
-- [Darkempire78/Github1s-Extension](https://github.com/Darkempire78/Github1s-Extension)
-- [zvizvi/Github Web IDE](https://github.com/zvizvi/github-web-ide) - [Chrome Web Store](https://chrome.google.com/webstore/detail/adjiklnjodbiaioggfpbpkhbfcnhgkfe)
+- [Darkempire78/GitHub1s-Extension](https://github.com/Darkempire78/GitHub1s-Extension)
```

Comment:
GitHub repository names are case preserving, but not case sensitive, so this can be done even if @darkempire78 doesn't rename their repository.